packages/boot/src/__tests__/integration/interceptor.booter.integration.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  BindingScope,
+  ContextTags,
+  GLOBAL_INTERCEPTOR_NAMESPACE,
+} from '@loopback/core';
+import {expect, TestSandbox} from '@loopback/testlab';
+import {resolve} from 'path';
+import {BooterApp} from '../fixtures/application';
+
+describe('global interceptor script booter integration tests', () => {
+  const SANDBOX_PATH = resolve(__dirname, '../../.sandbox');
+  const sandbox = new TestSandbox(SANDBOX_PATH);
+
+  let app: BooterApp;
+
+  beforeEach('reset sandbox', () => sandbox.reset());
+  beforeEach(getApp);
+
+  it('boots global interceptors when app.boot() is called', async () => {
+    const expectedBinding = {
+      key: `${GLOBAL_INTERCEPTOR_NAMESPACE}.MyGlobalInterceptor`,
+      tags: [
+        ContextTags.PROVIDER,
+        ContextTags.TYPE,
+        ContextTags.GLOBAL_INTERCEPTOR,
+        ContextTags.NAMESPACE,
+        ContextTags.GLOBAL_INTERCEPTOR_GROUP,
+      ],
+      scope: BindingScope.TRANSIENT,
+    };
+
+    await app.boot();
+
+    const bindings = app
+      .findByTag(ContextTags.GLOBAL_INTERCEPTOR)
+      .map(b => ({key: b.key, tags: b.tagNames, scope: b.scope}));
+    expect(bindings).to.containEql(expectedBinding);
+  });
+
+  async function getApp() {
```

Comment:
Please improve the name to communicate that this is not just any application, but it's an application with an interceptor artifact.